docs/source/chatops/aliases.rst

```diff
@@ -133,6 +133,13 @@ In this the action google.get_search_results would be called with the parameters
        query: StackStorm
        limit: 10
 
+Additional ChatOps parameters passed to the command
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An execution triggered via chatops will contain variables such as ``action_context.api_user``, ``action_context.user`` and ``action_context.source_channel``. ``api_user`` is the user who's kicking off the chatops command from
```

Comment:
Opened a ticket for a future improvement https://github.com/StackStorm/st2/issues/2156

For now, that works and what manas has said - thanks for the contribution!